hardware/Utils/tests.py

```diff
@@ -0,0 +1,12 @@
+import unittest
```

Comment:
Here is the test directory path `hardware/tests`. We can move it there.

Reply:
Will do it